scripts/argocd-helper.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+#
+# Tests if the ArgoCD instance is available on the cluster by logging in.
+#
+# Uses the ArgoCD session, created by previsoly running "argocd login", to
+# generate a acccount token. The informaton is then stored on the ARGOCD_ENV_FILE
```

Comment:
a acccount -> an account
stored on -> stored in

Reply:
Also fixed 🙏